components/update_client/BUILD.gn

```diff
@@ -246,6 +246,8 @@ target(gtest_target_type, "cobalt_slot_management_test") {
     "//starboard/loader_app:installation_manager",
     "//testing/gmock",
     "//testing/gtest",
-    "//starboard/loader_app"
+
+    # TODO: b/296919440 - Make this a data dep 
```

Comment:
nit: please use the `TODO(b/296919440): ...` format.

go/cobalt-style#todo-format

Reply:
Never mind - as you pointed out offline, Google's internal style guidance for TODO formats has changed.

Reply:
resolved